pkg/receiver/gitproviderreceiver/README.md

```diff
@@ -108,3 +114,39 @@ The available scrapers are:
 |----------|-------------------------|
 | [github] | Git Metrics from [GitHub](https://github.com/) |
 | [gitlab] | Git Metrics from [GitLab](https://gitlab.com) |
+
+## Rate Limiting
+
+Given this receiver scrapes data from Git providers it is subject to rate
+limiting. The following sections will give some sensible defaults for each
+git provider.
+
+### GitHub
+
+This receiver primarily interacts with GitHub's GraphQL API. The default rate
+limit for GraphQL API is 5,000 points per hour (unless your PAT is associated
+to a GitHub Enterprise Cloud organization then your limit is 10,000).
+The receiver on average costs 4 points per repository. This means that the
+receiver can at most scrape 1250 repositories per hour.
+
+Given this average cost a good collection interval in seconds is
+
+```go
+(repository_count * 4 / hourly_graphql_rate / 3600) + 300
```

Comment:
```suggestion
(repository_count * 4 / hourly_graphql_rate / 3600) + 300
```

Thinking we can turn lines 134 -> 136 into a math expression. Maybe something like 

$` (n*4/r/3600)+300 `$

Where: 
n = number of repositories
r = hourly rate (5,000 or 10,000)